configs/roboraptor_24hu/roboraptor_24hu_BASE.xml

```diff
@@ -0,0 +1,125 @@
+                    <resp>creator</resp>
+                    <orgName>ELTE-DH<ref type="http://elte-dh.hu"/>
+                    </orgName>
+                </respStmt>
+                <respStmt>
+                    <resp>project director</resp>
+                    <persName><surname>Palkó</surname><forename>Gábor</forename>
+                        <ref>https://orcid.org/0000-0002-4394-8577</ref></persName>
+                </respStmt>
+                <respStmt>
+                    <resp>chief programmer</resp>
+                    <persName><surname>Indig</surname><forename>Balázs</forename>
+                        <ref>https://orcid.org/0000-0001-8090-3661</ref></persName>
+                </respStmt>
+                <respStmt>
+                    <resp>TEI expert</resp>
+                    <persName><surname>Fellegi</surname><forename>Zsófia</forename>
+                        <ref>https://orcid.org/0000-0001-9199-1759</ref></persName>
+                </respStmt>
+                <respStmt>
+                    <resp>programmer</resp>
+                    <persName><surname>Sárközi-Lindner</surname><forename>Zsófia</forename>
+                        <ref>https://orcid.org/0000-0002-2558-0633</ref></persName>
+                </respStmt>
+            </editionStmt>
+            <publicationStmt>
+                <publisher>
+                    <orgName>ELTE-DH</orgName>
+                    <ref type="url">http://elte-dh.hu/</ref>
+                </publisher>
+                <pubPlace>Budapest <ref type="url">http://www.geonames.org/3054643</ref>
+                </pubPlace>
+                <date>2020</date>
+                <availability>
+                    <p>Metadata: IN COPYRIGHT - NON-COMMERCIAL USE PERMITTED<ref type="url"
+                        >http://rightsstatements.org/vocab/InC-NC/1.0/</ref></p>
+                    <p>Text: IN COPYRIGHT <ref type="url"
+                        >http://rightsstatements.org/vocab/InC/1.0/</ref>
+                    </p>
+                </availability>
+                <idno type="PID"></idno>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title></title>
+                    <publisher><orgName>Central Médiacsoport</orgName>
+                        <placeName>1037 Budapest, Montevideo u. 9.</placeName>
+                        <ref type="url" source="https://24.hu/impresszum/"><!--KITÖLT portál DOI-ja pl https://doi.org/10.5281/zenodo. --></ref>
+                        <date when="2021-09-07"/>
+                    </publisher>
+                    <pubPlace>
+                        Budapest
+                        <ref type="url">http://www.geonames.org/3054643</ref>
+                    </pubPlace>
+                    <availability><p>Copyright © 2021 Central Médiacsoport Zrt.</p>
+                        <p><ref type="url" source="https://24.hu/felhasznalasi-feltetelek/"><!--KITÖLT https://doi.org/--></ref>-->
```

Comment:
Please remove the `-->` from the end of this line. That might result in invalid xml-s.

Reply:
That is a typical issue in the portal specific `_BASE.xmls`. Please correct that in all relevant files.